generate_boilerplate.py

```diff
@@ -0,0 +1,99 @@
+import glob
```

Comment:
I'll rename the file to something like `generate_apps_package` and put it under `scripts` directory